src/tests/ftest/util/job_manager_utils.py

```diff
@@ -484,14 +482,13 @@ def assign_environment(self, env_vars, append=False):
             # dictionary keys with the specified values or add new key value
             # pairs to the dictionary.  Finally convert the updated dictionary
             # back to a string for the parameter assignment.
-            original = EnvironmentVariables({
-                item.split("=")[0]: item.split("=")[1] if "=" in item else None
-                for item in self.export.value.split(",")})
+            original = EnvironmentVariables.from_list(
+                [item for item in self.export.value.split(",")])
```

Comment:
(pylint-unnecessary-comprehension) Unnecessary use of a comprehension, use list(self.export.value.split(',')) instead.